docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+---
+mongo:
+  image: mongo:2.6
+content:
+  image: quay.io/deconst/content-service
+  links:
+  - "mongo:mongo"
+  environment:
```

Comment:
Something secret (by which I mean I forget to put it in the README): if you run the content-service with `STORAGE=memory`, all of the `RACKSPACE_`, `MONGODB`, and container variables are optional and the service stores everything in-memory. If you're just using this to have an endpoint to submit stuff, that might be useful for you here.